backend/http/http.go

```diff
@@ -365,6 +378,36 @@ func parse(base *url.URL, in io.Reader) (names []string, err error) {
 	return names, nil
 }
 
+type GoIndexFile struct {
+	Id           string `json:"id"`
+	MimeType     string `json:"mimeType"`
+	ModifiedTime string `json:"modifiedTime"`
+	Name         string `json:"name"`
+	// don't know why, but this is a string instead of an int64 (against gdrive doc)
+	Size          string `json:"size"`
+	ThumbnailLink string `json:"thumbnailLink"`
```

Comment:
AFAIK ThumbnailLink is unneeded and can be discarded, but left in the chance that rclone can actually use it (if not I'll remove)